recipes/scalene/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set package_name = "scalene" %}
+{% set initial = "s" %}
+{% set version = "1.5.7" %}
+{% set org = "plasma-umass" %}
+{% set author = "emeryberger" %}
+
+package:
+  name: {{ package_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ package_name }}/archive/refs/tags/v{{ version }}.tar.gz
+  # See https://pypi.org/project/scalene/#files
+  sha256: 2d77a1d5f2a1b9314e64e9ec97d068e6e2e0cb74a28f219924477c8b251b806a
+
+build:
+  skip: True  # [py<38]
+  number: 0
```

Would suggest just using `script` and dropping `build.sh` & `bld.bat`. This line is [taken from the example]( https://github.com/conda-forge/staged-recipes/blob/66111e0d8cb287af412285f00a81a1cf9fdd67f5/recipes/example/meta.yaml#L38 )

```suggestion
  number: 0
  script: {{ PYTHON }} -m pip install . -vv
```